guides/common/modules/proc_customizing-registration-by-using-snippets.adoc

```diff
@@ -0,0 +1,23 @@
+[id="customizing-registration-by-using-snippets"]
+= Customizing registration by using snippets
+
+You can customize the registration process without editing the *Global Registration* template by creating snippets that the template includes if they exist.
+To add custom steps to registration, create one or both of the following snippets:
+
+* `before_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the beginning of registration.
+* `after_registration` {endash} This snippet will be loaded and executed by the *Global Registration* template at the end of registration.
+
+You have to name the snippets precisely.
+Otherwise, they cannot be recognized.
+
+.Prerequisites
+* Your {Project} account has a role that grants the `view_provisioning_templates` and `create_provisioning_templates` permissions.
+* You have to select a particular organization and location context.
```

Comment:
This feels weird to me. a) it's almost always required to select a context; b) you have to assign the snippets to a specific org/loc context.

Reply:
```suggestion
* Select a particular organization and location context.
```
or
```suggestion
* A particular organization and location context have been selected.
```
I think the second one would be better in terms of parallelism, but it's also much longer :shrug:

Reply:
How about "You have selected..."?